raiden/transfer/log.py

```diff
@@ -85,6 +85,30 @@ def __init__(self, database_path):
         )
         self.conn.commit()
 
+        self.sanity_check()
+
+    def sanity_check(self):
+        """ Ensures that NUL character can be safely inserted and recovered
```

Comment:
The character is [abbreviated `NUL`](https://en.wikipedia.org/wiki/Null_character), what is the problem with it?